deployment/base/rbac/master-clusterrole.yaml

```diff
@@ -22,3 +22,18 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  verbs:
+  - create
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  resourceNames:
+  - "nfd-master.nodefeaturediscoveries.nfd.kubernetes.io"
```

Comment:
nit: not that the name would really matter but there seems to be some unnecessary repeating/tautology here (nodefeaturediscoveries.nfd), how about making it a tad shorter like `nfd-master.nfd.kubernetes.io`, WDYT?

TBH, I have no idea if there are some best practices on naming these...

Reply:
I have done it in a manner how node-feature-operator defines lock id:
https://github.com/kubernetes-sigs/node-feature-discovery-operator/blob/master/main.go#L99
I am open to changing it :)

Reply:
Ach, of course. `nodefeaturediscoveries` is the name of the CRD. In this scheme `nfd-master.nfd.kubernetes.io` would be more logical, IMO. Any thoughts @ArangoGutierrez?

Reply:
changed to `nfd-master.nfd.kubernetes.io`